src/services/pricing-engine.ts

```
@@ -0,0 +1,242 @@
+  const { baseBwPrice, baseColorPrice, colorMultiplier } = config.pricing;
+  const { blankPagePolicy } = config;
+
+  switch (classification) {
+    case 'blank':
+      return blankPagePolicy === 'charge_zero' ? 0 : baseBwPrice;
+    case 'bw':
+      return baseBwPrice;
+    case 'full_color':
+      return baseColorPrice;
+    case 'partial':
+      const rawPrice = baseBwPrice + coverage * colorMultiplier;
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)